resources/artifact-ignores.properties

```diff
@@ -106,6 +106,7 @@ HiddenParameter                  # renamed to hidden-parameter
 hipchat-plugin                   # renamed to hipchat
 # service discontinued -- https://wiki.jenkins.io/display/JENKINS/HockeyApp+Plugin
 hockeyapp = https://wiki.jenkins.io/x/7oPPAw
+hpe-network-virtualization-plugin	# end of life
```

Comment:
Could you provide a reference for why this plugin is deprecated? If so, that URL should be the value to this key, as it will be provided to users of the plugin as an explanation.

Otherwise there will not be a deprecation notice at all, the plugin will just be gone.

Reply:
```suggestion
hpe-network-virtualization-plugin = https://github.com/jenkins-infra/update-center2/pull/615#discussion_r913923468
```

😁

Reply:
I am not sure my managers will love this 😂. Everything a customer sees has to go through technical writer. I prefer to leave it as is.

Reply:
There is no solution we support in which the plugin would be suspended from distribution AND marked deprecated without pointing to a custom URL. The reason is that suspending distribution removes its page from plugins.jenkins.io, and there's no way to get anywhere without having a URL. WIth a URL, there'll be a deprecation notice both inside Jenkins and on plugins.jenkins.io, like https://plugins.jenkins.io/xtrigger/

I'd usually suggest a one line notice in the project's readme and to link there, but this plugin is still on the read-only wiki, and would need a new release to switch the documentation source to GitHub.

Does the related product have a deprecation/EOL notice we could hijack for the Jenkins plugin integrating with it?

Reply:
I don't think so. I am not sure when exactly this product reached its EOL. But, the trigger for the plugin removal were many security issues discovered by the Jenkins security team. One of my managers was notified about it. Since I am the only contributor/owner of the plugin, I was asked to remove it.

Reply:
I have a link about the EOL of our stand alone product. I will make the changes

Reply:
I added a URL. Please approve and merge